specs/NonClientHitTestKind.md

```diff
@@ -198,11 +253,28 @@ namespace Microsoft.Web.WebView2.Core {
         NonClientRightButtonUp = 0x00A5,
         ...
     }
+    runtimeclass CoreWebView2NonClientRegionChangedEventArgs
+    {
+        // ICoreWebView2NonClientRegionChangedEventArgs members
+        CoreWebView2NonClientRegionKind Region { get; };
+
+    }
+    runtimeclass CoreWebView2RegionRectCollection
```

Comment:
Shouldn't need this runtime class for C#/WinRT

Reply:
removed